### PR TITLE
Pr/voice out functionality

### PR DESCRIPTION
**Pull request by Amaan1234567**
Insert description of pull request here

Checklist(to mark the completed tasks, replace the space inside the brackets with an x):
- [x] Are the variable and function named correctly. Example: ```myNewVariable```
- [x] Are the class and structs named correctly. Example: ```MyNewClass```
- [x] Are the comments clear and concise.
- [x] Is the documentation updated.
- [x] There are no new warnings than before*.
- [x] Are the logs used correctly. Only logs are allowed. No print statements.

*The only reason new warnings may be allowed is if the code that produces the warnings are from third party code that we cannot modify.(Example: Qt, KDE Frameworks, git submodules etc.)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.20)
 
 # Project definition
-project(KoolIntelligence VERSION 0.0.0 LANGUAGES CXX)
+project(KoolIntelligence VERSION 0.0.0 LANGUAGES CXX C)
```
Comment (RahulVadhyar) [inline CMakeLists.txt]: not needed, remove this

File: src/model_backend/thirdparty/CMakeLists.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ set(CMAKE_CUDA_COMPILER "/usr/local/cuda/bin/nvcc" CACHE FILEPATH "CUDA compiler
 
 set(BUILD_SHARED_LIBS OFF)
 find_package(SDL2 REQUIRED)
-enable_language(CUDA)
+#enable_language(CUDA)
```
Comment (RahulVadhyar) [inline src/model_backend/thirdparty/CMakeLists.txt]: enable this

File: src/model_backend/tts_service.hpp [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#include <cstdlib>
+#include <filesystem>
+
+class TTSService
+{
+public:
+    TTSService()
+    {
```
Comment (RahulVadhyar) [inline src/model_backend/tts_service.hpp]: this should not be there in hpp

**Review by RahulVadhyar — CHANGES_REQUESTED** [review]

File: src/model_backend/thirdparty/CMakeLists.txt (hunk 2 above)
Comment (Amaan1234567) [reply src/model_backend/thirdparty/CMakeLists.txt]: its not needed

**Review by shrenisc — APPROVED** [review]
Approved

File: CMakeLists.txt (hunk 1 above)
Comment (Amaan1234567) [reply CMakeLists.txt]: yea

**Review by RahulVadhyar — APPROVED** [review]

**Pull request merged**